### ISSUE TITLE
Unable to load custom font using satori options

### ISSUE THREAD

**Issue by Bumbleboss** [harlan-zw/nuxt-og-image#42]
### Describe the bug

In `nuxt.config.ts`
```ts
ogImage: {
  fonts: ['Manrope:500'],
  satoriOptions: {
    fonts: [{
      name: 'optieinstein',
      style: 'normal',
      weight: 800,
      data: fs.readFileSync('./public/fonts/OPTIEinstein_black.woff'),
    }]
  },
}
```
Above code does not add a custom font. Instead, I am met with this system log
```
seo-image unknown or invalid utility
```

How can one use custom fonts that are in the project directory and not Google Fonts-based?

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Windows_NT
- Node Version:     v18.15.0
- Nuxt Version:     3.4.2
- Nitro Version:    2.3.3
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      runtimeConfig, typescript, extends, ogImage, css, vite, modules, app
- Runtime Modules:  @nuxt/content@2.6.0, @nuxt/image-edge@1.0.0-28020728.5df24eb
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Hey @Bumbleboss, thanks for the issue.

Custom font loading wasn't supported before.

I've now implemented it in the latest beta, see the [Custom Fonts](https://github.com/harlan-zw/nuxt-og-image#custom-fonts--supporting-non-english-characters) documentation on implementation details.

**Comment by Bumbleboss** [comment]
This error keeps showing up despite following the documentation.
![image](https://user-images.githubusercontent.com/22979938/235311039-b36df52f-a86e-43c9-869e-6d6862ad267a.png)


Would it also be possible to have custom font *and* google font added together?
I am currently using 2.0.0-beta.23

**Comment by harlan-zw** [comment]
@Bumbleboss Would you share your config? I'm using the below and it's working as expected

```ts
  ogImage: {
    fonts: [
      'Inter:400',
      'Inter:700',
      {
        name: 'optieinstein',
        weight: 800,
        path: '/OPTIEinstein-Black.otf',
      }
    ],
  }
```

**Comment by Bumbleboss** [comment]
On version 2.0.0-beta.27. Error changed and image is no longer being rendered. The difference is `.woff`
```ts
  ogImage: {
    fonts: [
      'Manrope:500',
      {
        name: 'optieinstein',
        weight: 800,
        path: '/fonts/OPTIEinstein_black.woff',
      }
    ],
  }
```

This is what appears on terminal:
![image](https://user-images.githubusercontent.com/22979938/235353504-6cbbb77f-1087-4300-8218-fb70b20ad530.png)

And image itself is not being rendered, it becomes transparent.
![image](https://user-images.githubusercontent.com/22979938/235353533-a7b4169f-c6c9-4b9b-9ed3-80a40822b83a.png)

**Comment by harlan-zw** [comment]
Is the white screen and error only there when using the `.woff`? 

Also can you double check you're using the latest, in the newest version all json extractions have a try / catch with them :thinking:

**Comment by harlan-zw** [comment]
Actually I can replicate this atm, looking into a fix

**Comment by Bumbleboss** [comment]
Was able to reproduce issue. I even tried having the image only show a simple text but the issue still persists. You will need to test it locally since StackBlitz causes an issue with retrieving Manrope font.

https://stackblitz.com/edit/github-ocxs1d?file=pages/index.vue

Issue still exists after removing custom font, it was rendering fine on stable release (without using custom local font)

**Comment by harlan-zw** [comment]
Thanks for the reproduction!

This was an interesting bug, for whatever reason the `#` from your props wasn't being encoded properly so the json payload was being cut short.

Fixed in 2.0.0-beta.29

**Comment by Bumbleboss** [comment]
Can confirm, it works now! Thanks!
